app/view_models/workarea/storefront/search_autocomplete_view_model.rb

```diff
@@ -2,10 +2,16 @@ module Workarea
   module Storefront
     class SearchAutocompleteViewModel < ApplicationViewModel
       def products
-        return [] if searches.blank?
```

Comment:
I think we'd still want this? This was here to short-circuit running the whole query (pretty expensive) if we know that there aren't any searches to match on.